src/k8s-engine/wsl.ts

```diff
@@ -221,6 +221,54 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     }
   }
 
+  /**
+   * Use wslpath to convert windows path to the native wsl path
+   */
+  protected async wslify(windowsPath: string): Promise<string> {
+    // Convert '\'s to '/'s or they'll get dropped before wslpath sees them.
```

Comment:
I haven't been able to verify this -- since upgrading my main laptop to Big Sur, I can't run anything on windows, but before I definitely was getting file-not-found errors because single backslashes were getting stripped by the WSL subsystem.

Reply:
Backslash-escaping isn't needed when not going through a shell command-line expansion/escaping thing -- backslashes make it to the windows `wslpath` executable as is.